examples/mesh_viewer/Main.cc

```diff
@@ -66,7 +66,16 @@ void buildScene(ScenePtr _scene)
   MeshPtr meshGeom = _scene->CreateMesh(descriptor);
   mesh->AddGeometry(meshGeom);
   root->AddChild(mesh);
-//! [create a mesh]
+
+//! [create a glb pbr mesh]
+  mesh = _scene->CreateVisual();
+  mesh->SetLocalPosition(3, 2, 0);
+  mesh->SetLocalRotation(0, 0, 0);
+  descriptor.meshName = common::joinPaths(RESOURCE_PATH, "AmbulanceStretcher.glb");
+  descriptor.mesh = meshManager->Load(descriptor.meshName);
+  meshGeom = _scene->CreateMesh(descriptor);
+  mesh->AddGeometry(meshGeom);
+  root->AddChild(mesh);
```

Comment:
add another `//! [create a glb pbr mesh]` here?

Alternatively remove the one above since it does not look like we are referring to it in the tutorial.

Reply:
Done! https://github.com/gazebosim/gz-rendering/pull/642/commits/1fe7958360432922e3faecdf4fe7565d3cd28a8c